src/common/com/intellij/plugins/haxe/lang/psi/impl/AnonymousHaxeTypeImpl.java

```diff
@@ -39,9 +36,12 @@ public AnonymousHaxeTypeImpl(@NotNull ASTNode node) {
   @NotNull
   @Override
   public List<HaxeType> getHaxeExtendsList() {
```

Comment:
For typedef sometimes it will be list of [extends]([url](https://haxe.org/manual/type-system-extensions.html)). In other cases extends list will be contain just one entry.